src/main/java/org/gridsuite/explore/server/ExploreController.java

```diff
@@ -273,11 +273,12 @@ public ResponseEntity<Void> changeFilter(@PathVariable UUID id, @RequestBody Str
     public ResponseEntity<Void> updateContingencyList(
             @PathVariable UUID id,
             @RequestParam(name = "name") String name,
+            @RequestParam(QUERY_PARAM_DESCRIPTION) String description,
```

Comment:
I may be wrong but I don't see any reason to have it required. He may have a default value as an empty string in order to simplify `updateContingencyList` call. 
```suggestion
            @RequestParam(QUERY_PARAM_DESCRIPTION, required = false) String description,
```

Reply:
ok